projects/ppwcode/ng-router/src/lib/breadcrumb/breadcrumb.component.ts

```diff
@@ -0,0 +1,19 @@
+import { AsyncPipe } from '@angular/common'
+import { Component, inject } from '@angular/core'
+import { RouterLink } from '@angular/router'
+import { TranslatePipe } from '@ngx-translate/core'
+import { shareReplay } from 'rxjs'
+import { BreadcrumbProviderOptions } from './breadcrumb-provider-options.model'
+import { BREADCRUMB_PROVIDER_OPTIONS, BreadcrumbService } from './breadcrumb.service'
+
+@Component({
+    selector: 'ppw-breadcrumb',
+    templateUrl: './breadcrumb.component.html',
+    imports: [RouterLink, TranslatePipe, AsyncPipe],
+    styleUrls: ['./breadcrumb.component.scss']
+})
+export class BreadcrumbComponent {
+    breadcrumbService: BreadcrumbService = inject(BreadcrumbService)
+    public options: BreadcrumbProviderOptions = inject(BREADCRUMB_PROVIDER_OPTIONS)
```

Comment:
Make both protected and readonly, to prevent people from accessing them through the component.